packages/react/src/Bento/Bento.tsx

```diff
@@ -0,0 +1,248 @@
+
+type Align = 'start' | 'center' | 'end'
+type ResponsiveAlign = Partial<Record<Size, Align>>
+
+type Padding = 'condensed' | 'normal' | 'spacious'
+type ResponsivePadding = Partial<Record<Size, Padding>>
+
+type BentoProps = React.HTMLAttributes<HTMLDivElement> & BaseProps<HTMLDivElement>
+
+const Root = ({className, ...rest}: BentoProps) => {
+  return <section className={clsx(styles.Bento, className)} {...rest}></section>
+}
+
+type BentoItemProps = {
+  columnStart?: ColumnIndex | ResponsiveColumnIndex
+  columnSpan?: ColumnIndex | ResponsiveColumnIndex
+  rowStart?: number | ResponsiveRowIndex
+  rowSpan?: number | ResponsiveRowIndex
+  flow?: Flow | ResponsiveFlow
+  colorMode?: ColorModesEnum.LIGHT | ColorModesEnum.DARK
+  visualAsBackground?: boolean
+} & React.HTMLAttributes<HTMLDivElement> &
+  BaseProps<HTMLDivElement>
+
+const returnClassBasedOnResponsiveMap = (
+  classIdentifier: string,
+  propName: string,
+  prop?:
+    | ResponsiveColumnIndex
+    | ColumnIndex
+    | ResponsiveRowIndex
+    | number
+    | ResponsiveFlow
+    | Flow
+    | ResponsiveAlign
+    | Align
+    | ResponsivePadding
+    | Padding,
+) => {
+  const classesToMerge: string[] = []
+  if (typeof prop === 'number' || typeof prop === 'string') {
+    classesToMerge.push(styles[`${classIdentifier}--${propName}-${prop}`])
+  } else if (typeof prop === 'object') {
+    for (const [key, value] of Object.entries(prop)) {
+      classesToMerge.push(styles[`${classIdentifier}--${key}-${propName}-${value}`])
+    }
+  }
+  return classesToMerge
+}
+
+const Item = ({
+  className,
+  columnStart,
+  columnSpan,
+  rowStart,
+  rowSpan,
+  flow = 'row',
+  colorMode,
+  visualAsBackground = false,
+  children,
+  ...rest
+}: BentoItemProps) => {
+  const bentoItemClassArray = [styles.Bento__Item]
+  bentoItemClassArray.push(
+    ...returnClassBasedOnResponsiveMap('Bento__Item', 'column-span', columnSpan),
+    ...returnClassBasedOnResponsiveMap('Bento__Item', 'row-span', rowSpan),
+    ...returnClassBasedOnResponsiveMap('Bento__Item', 'column-start', columnStart),
+    ...returnClassBasedOnResponsiveMap('Bento__Item', 'row-start', rowStart),
+    ...returnClassBasedOnResponsiveMap('Bento__Item', 'flow', flow),
+  )
+
+  if (!visualAsBackground) {
+    flow === 'column' && bentoItemClassArray.push(styles['Bento-column-padding-override'])
+    flow === 'row' && bentoItemClassArray.push(styles['Bento-row-padding-override'])
+  }
+
+  const colorModeProp = colorMode ? {'data-color-mode': colorMode} : {}
+
+  return (
+    <div
+      {...colorModeProp}
+      className={clsx(
+        ...bentoItemClassArray,
+        visualAsBackground && styles[`Bento__Item--visual-as-background`],
+        className,
+      )}
+      {...rest}
+    >
+      {children}
+    </div>
+  )
+}
+
+type BentoContentProps = {
+  leadingVisual?: ReactElement | Icon
+  padding?: Padding | ResponsivePadding
+  verticalAlign?: Align | ResponsiveAlign
+  horizontalAlign?: Align | ResponsiveAlign
```

Comment:
I don't think Bento should support a `end` horizontal alignment. I can't find any designs that use it or mentions of it in the API exploration.

Reply:
+ 1 on this, no need of `end` here